aws/resource_aws_route.go

```diff
@@ -526,3 +541,72 @@ func findResourceRoute(conn *ec2.EC2, rtbid string, cidr string, ipv6cidr string
 		"you need to specify a CIDR block of IPv6 CIDR Block", rtbid)
 
 }
+
+func resourceAwsRouteImportState(
+	d *schema.ResourceData,
+	meta interface{}) ([]*schema.ResourceData, error) {
+
+	log.Printf("[INFO] Attempting to import aws_route, ID: %s", d.Id())
+	conn := meta.(*AWSClient).ec2conn
+
+	err := resourceAwsRouteCheckForImport(d)
+	if err == nil {
+		routeTableID := d.Get("route_table_id").(string)
+		cidr := d.Get("destination_cidr_block").(string)
+		ipv6cidr := d.Get("destination_ipv6_cidr_block").(string)
+		route, err := resourceAwsRouteFindRoute(conn, routeTableID, cidr, ipv6cidr)
+
+		if err == nil {
+			results := make([]*schema.ResourceData, 1)
+			resourceAwsRouteSetResourceData(d, route)
+			results[0] = d
+			return results, nil
+		}
+	}
+
+	return nil, err
+
+}
+
+// AWS does not provide an id for routes in route tables. To import, use a pseudo
+// ID of r-ROUTETABLEID-DESTINATION (e.g., r-rtb-12345678_10.2.0.0/16). This
```

Comment:
_ not - as a separator, no? i.e. `r-ROUTETABLEID_DESTINATION` not `r-ROUTETABLEID-DESTINATION`....